examples/server_async.py

```diff
@@ -286,8 +286,9 @@ def get_commandline():
         args.slaves = 0
     if not args.framer:
         args.framer = comm_defaults[args.comm][0]
-    if not args.port:
-        args.port = comm_defaults[args.comm][1]
+    args.port = args.port or comm_defaults[args.comm][1]
```

Comment:
This is quite a complicated way of doing that. please use the comm_defaults where the transport defaults are stored.

Reply:
I don't see any complication here, use `port` if passed or use the comm defaults. What is so complicated here ?